api/v1.inc

```diff
@@ -30,7 +30,11 @@ $router->add_route("GET", "v1/projects/specialdays", "api_v1_projects_specialday
 $router->add_route("GET", "v1/projects/imagesources", "api_v1_projects_imagesources");
 $router->add_route("GET", "v1/projects/:projectid/pages", "api_v1_project_pages");
 $router->add_route("GET", "v1/projects/:projectid/pagedetails", "api_v1_project_pagedetails");
-$router->add_route("GET", "v1/projects/:projectid/pages/:pagename/pagerounds/:pageroundid", "api_v1_project_page_round");
+$router->add_route("GET", "v1/projects/:projectid/pages/:pagename/pagerounds/:pageroundid", "api_v1_project_page_pageround");
+$router->add_route("GET", "v1/projects/:projectid/codepoints", "api_v1_project_codepoints");
+
+$router->add_route("POST", "v1/projects/:projectid/rounds/:roundid", "api_v1_project_round");
+$router->add_route("PUT", "v1/projects/:projectid/pages/:pagename/rounds/:roundid", "api_v1_project_page_round");
```

Comment:
The whole pageround/round I created is a little weird (thanks OCR "round") but I think we should make this a PUT against the same `v1/projects/:projectid/pages/:pagename/pagerounds/:pageroundid` URL. You can have it as a separate function still and use the `:roundid` to do the round validation for the PUT separately than the GET though.

Reply:
I'm not sure about this. These are rounds that the user can work in. But OCR is not really the same sort of thing.

Reply:
I agree it's odd, but it fits better with the RESTful notion that a URL uniquely defines a resource to work with. Having one URL to get the page and another to save the page is weirder than having a round that you can only get and not set.

Reply:
I don't think we can use the same GET function that we use for page browse because we want to check that the user can get the text for proofreading. We can use Lpage->resume saved page first, which just does some checks if the page is 'out' or 'temp' and also changes its state to 'temp' if it was 'saved'.

Reply:
How what I've made now uses the 'open' action to get the text.